week2/homework/src/update.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/** ---------------------------------------
+     Base
+ ---------------------------------------- */
+const fs = require('fs');
+
+/** ---------------------------------------
+     Function for Command
+ ---------------------------------------- */
+const update = {
+  execute: (toDoNumber, updatedToDo) => {
+    const content = fs.readFileSync('To-Do Database.json', 'utf8');
+    const toDoIndex = toDoNumber - 1;
+    switch (content) {
+      case '':
+        console.log('There is nothing to update !');
+        break;
+      default:
+        fs.readFile('To-Do Database.json', 'utf8', (err, readData) => {
+          if (err) throw err;
+          const parsedJSON = JSON.parse(readData);
+          const arrayLength = parsedJSON.length;
+          if (toDoNumber < 0 || toDoNumber > arrayLength - 1) {
```

Comment:
Watch out about this bit: `arrayLength - 1`.
What happens if I have only 1 item in the list, and I try to do:
node index.js update 1 "Random string"

Reply:
Yes. It's been corrected.